modules/core/shared/src/main/scala/lucuma/core/enums/ImageQuality.scala

```diff
@@ -21,7 +21,7 @@ sealed abstract class ImageQuality(val tag: String, val toDeciArcSeconds: Quanti
   def label: String        = f"""< ${toArcSeconds.value.toDouble}%.1f\""""
 
   def toAngle: Angle =
-    Angle.fromMicroarcseconds(toDeciArcSeconds.tToUnit[MicroArcSecond].value.value.toLong)
+    Angle.fromMicroarcseconds(toDeciArcSeconds.value.value * 100_000L)
```

Comment:
If anybody understands coulomb and feels sufficient anxiety about bypassing it, I'd welcome input on how the conversion stuff is supposed to work.  I'm happy with just making it work though.

Reply:
@armanbilge Can you enlighten us?